otelib/client.py

```diff
@@ -13,74 +15,12 @@ class OTEClient:
 
     """
 
-    def __init__(self, url: str) -> None:
+    def __new__(cls, url: str):
```

Comment:
Why use `__new__` rather than `__init__` here?

Reply:
__init__ isn't supposed to have a return, while __new__ can. If we decide to go with a combined parent class for the Services/Python-Client, this could be an important aspect to think about

Reply:
https://github.com/EMMC-ASBL/otelib/issues/68